install/tools/man/ipa-cacert-manage.1

```diff
@@ -72,6 +74,13 @@ Please do not forget to run ipa-certupdate on the master, all the replicas and a
 .RS
 Display a list of the nicknames or subjects of the CA certificates that have been installed.
 .RE
+.TP
+\fBprune\fR
+\- Prune the stored CA certificates
+.sp
+.RS
+Removes installed CA certificates that are not longer valid.
```

Comment:
I think it would be clearer to say that it removes CA's that are expired. "not longer valid" could include other reasons like revocation.